routes/operation.js

```diff
@@ -103,12 +103,17 @@ router.param('id', asyncHandler(async function (req, res, next) {
   next();
 }));
 
-router.get('/:id', function (req, res) {
-  res.json(assign(pick(req.operation, ATTRIBUTES), {
-    data: req.operation.data.toString('hex'),
-    signature: req.operation.signature.toString('hex'),
+router.get('/:id', asyncHandler(async function (req, res) {
+  const operation = await Operation.findOne({
```

Comment:
`req.operation` was loaded  here: https://github.com/nguyenkha/mpc-demo/blob/48073f82000ea40c325a4a0e32bd59c30fb2688d/routes/operation.js#L92

Reply:
Resolved in https://github.com/nguyenkha/mpc-demo/pull/2/commits/6b4cf4d76e48324da65fdb65ff6f09ca4c7d6198